core/app/mailers/spree/base_mailer.rb

```diff
@@ -10,9 +9,19 @@ def money(amount, currency = Spree::Config[:currency])
     end
     helper_method :money
 
-    def mail(headers={}, &block)
+    def mail(headers = {}, &block)
+      ensure_default_action_mailer_url_host
       super if Spree::Config[:send_core_emails]
     end
 
+    private
+
+    # this ensures that ActionMailer::Base.default_url_options[:host] is always set
+    # this is only a fail-safe solution if developer didn't set this in environment files
+    # http://guides.rubyonrails.org/action_mailer_basics.html#generating-urls-in-action-mailer-views
+    def ensure_default_action_mailer_url_host
```

Comment:
Oh this's pretty nice :) But what do you think  about raise an exception with an useful message to the developer need to do this config ?